# README For Bioinformatics Final Project
## Developed by Maggie Walsh
## On April 19, 2021
## mcwalsh@dons.usfca.edu

### Overview
My plan for this project is to explore SARS-CoV-2 sequence data from New York City in the early stages of the 
Pandemic. I plan to use transportation data and other municipal data recources to track how the shelter in place 
order and other restrictions may have had an impact on the types of variants that were detected in the city 
around that time.

### SRA Bioproject ID
Acession: PRJNA721724 
ID number: 721724

### Outline of Analysis Steps
1. Sequence Analysis and Variant Calling
 * do a sequence analysis and call variants
 * organize variant data by date, look at rate of variants, different types, etc
2. Analysis of Tabular Data
 * use tabular data to visualize the timeline for coronavirus restrictions
3. Compare Variant and Restriction Data
 * create figures that demonstrate how coronavirus rescticions impacted variant data in New York

## Change Log
2021-4-26: Finished variant calling analysis of New York SARS-CoV-2 sequences. Added vcf data to report Rmd and started using NY covid data to create figures.
2021-4-19: Forked and cloned template repo, and updated README to include plan for final project. 


Parts of this pipeline approach are based on the pipeline described in the [Data Carpentry Genomics lessons](https://datacarpentry.org/genomics-workshop/), which are made available under a [CC-BY 4.0 license](https://creativecommons.org/licenses/by/4.0/).

