---
title: 'Analysis of SARS-CoV-2 Infection in New York City in the Early Stages of the COVID-19 Pandemic'
author: "Maggie Walsh"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output: pdf_document
csl: bioinformatics.csl
bibliography: references.bib
params:
  gff_file_path: "data/metadata/sars_refgenome_annotation.gff"
  vcf_dir_path: "data/11_vcf_output_for_R"
  sra_runtable_path: "data/00_sra_runtable/SraRunTable_NY_fixed.txt"
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE,
                      warning = FALSE,
                      message = FALSE,
                      error = TRUE)
```

# Background and Overview

In early 2020, the world was forever changed by the spread of the coronavirus disease (COVID-19), which was declared a global pandemic by the WHO by early March. In the United States, New York City became the epicenter, with 203,000 confirmed cases between the months of March and May [@thompson2020covid]. Although news of the virus was widely overlooked initially, people quickly became aware of the insidious nature of the SARS-CoV-2 virus with variable incubation times, and it's ability to spread via asymptomatic carriers. New York City was placed under a statewide lock-down on March 22, 2020, and the number of confirmed cases peaked a week later [@thompson2020covid]. As the machinery of our society was grinding to a halt, researchers raced to understand the SARS-CoV-2 virus in order to prevent further devastation. My goal for this report was to synthesize some of that research, and generate my own analysis that explains what was going on in New York City during the onset of the pandemic. My analysis started with a look into SARS-CoV-2 case numbers in New York, as well as a look at changes in population mobility during this time. I also used a bash pipeline to analyze sequence data taken from nasopharyngeal swabs of people in New York City from early March to late April. Twelve samples were sequenced and variants were called against a reference genome to determine SNPs for variant analysis. My findings support 

# Methods

## Sample Sequencing and Variant Calling
I downloaded sequences to a remote server from the [NCBI database](https://www.ncbi.nlm.nih.gov/) using the [accession ID](https://www.ncbi.nlm.nih.gov//bioproject/PRJNA721724), utilizing [fasterq-dump](https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump) from the NCBI SRA toolkit. Samples were trimmed using the trimmomatic tool [@bolger2014trimmomatic]. My workflow utilized the Burrows Wheeler Aligner (BWA) to map reads against the refrence genome [@li2009fast] and SAMtools and BCFtools to sort and process the mapped reads [@li2009sequence]. Specific steps of my pipeline can be found in the [code folder](https://github.com/walshmags/walshmags-bioinformatics-final-project/tree/main/code) of this repository. Parts of the pipeline approach are based on the pipeline described in the [Data Carpentry Genomics lessons](https://datacarpentry.org/genomics-workshop/), which are made available under a [CC-BY 4.0 license](https://creativecommons.org/licenses/by/4.0/). I executed the bash pipeline on the USF server using a makefile to output vcf files for analysis.

## Analysis

### Reading in vcf files
I analyzed the output from the bash pipeline in RStudio using a series of functions designed to format the vcf files into tabular data. The vcf files were read into R using the vcfR package [@knaus2017vcfr].

### Analysis of tabular data
I manipulated tabular data using the [dplyr](https://dplyr.tidyverse.org/) package. I made figures using the package [ggplot2](https://ggplot2.tidyverse.org/reference/index.html) and formatted them using the packages [ggthemes](https://cran.r-project.org/web/packages/ggthemes/index.html) and [gghighlight](https://cran.r-project.org/web/packages/gghighlight/vignettes/gghighlight.html).

### Additional data sources
I accessed additional data sources using the COVID19 R package [@Guidotti2020]. Case numbers originated from the Oxford [COVID-19 GOVERNMENT RESPONSE TRACKER](https://www.bsg.ox.ac.uk/research/research-projects/covid-19-government-response-tracker) dataset. Mobility data was from the [Google Mobility Reports](https://www.google.com/covid19/mobility/). I also used clade definitions from the [NextClade](https://clades.nextstrain.org/) project.

# Results and Discussion

```{r load-packages-and-functions}
library("vcfR")
library("ggplot2")
library("dplyr")
library("ggthemes")
library("COVID19")
library("gghighlight")
library("lubridate")
library("readr")

# this loads all function files in the code/functions directory
function_paths <- list.files("code/functions", full.names = TRUE)
purrr::walk(function_paths, source)
```

```{r load-vcf-data}
# load in, tidy, and stack all of the VCF files in the target directory
# these are the output of the bash pipeline
stacked_vcfs <- parse_tidy_and_stack_vcfs(
  vcf_dir_path = params$vcf_dir_path)

# load in the gff file with genome annotations for SARS-CoV-2
gff <- read_gff(gff_file_path = params$gff_file_path)

# pull out just the named genes from the annotation file
gene_table <- extract_genes_from_gff(annotation_object = gff)

# combine the stacked vcf table with the gene names and the metadata from
# the sra runtable file
vcf_with_metadata <- add_genes_metadata_to_vcfstack(
  sra_runtable_path = params$sra_runtable_path,
  stacked_vcf = stacked_vcfs,
  cleaned_genes_table = gene_table)
```

# Figures

```{r sample-plot}
# create a plot of unique SNP locations within each gene across all samples
vcf_with_metadata %>%
  filter(!is.na(gene)) %>% # get rid of SNPs not in gene regions
  group_by(gene, pos) %>%
  tally() %>% # this gives a column n for the number of gene by position
  group_by(gene) %>%
  tally() %>% # this collapses that down to the number of unique SNP locations
  ggplot(aes(x = gene,
             y = n)) +
    geom_col() +
    labs(title = "Count of distinct SNPs in Named SARS-CoV-2 Genes",
         x = "Gene Name") +
  theme_few() # get rid of the grey background
```

**Figure 1**: N and S genes have more unique SNPs in the set of samples analyzed.

```{r Covid-Data}
#trying to use COVID19 package to look at data regarding New York city
# first get all covid data from period of interest
covid_data_broad <- COVID19::covid19(country = "USA",
                                  level = 2,
                                  start = "2020-02-01",
                                  end = "2021-04-28")

# make a plot summarizing covid cases by state during this time
covid_plot <- covid_data_broad %>%
  ggplot(aes(date, confirmed)) +
  geom_line(aes(group = administrative_area_level_2)) +
  gghighlight(administrative_area_level_2 == "New York") +
  labs(title = "Number of Confirmed SARS-CoV-2 Cases by State",
       x = "Date", y = "Confirned Cases") +
  theme_clean()

covid_plot

```
***Figure 1***: Plot of confirmed SARS-CoV-2 cases in the past year. New York state was the hardest hit early on in the pandemic having a majority of the nations confirmed cases. The steep rise in cases was contained in subsequent months. In recent winter months, New York, alongside other states, has again seen a steep rise in cases.


```{r NY-Data}
# get specific covid data
covid_data_specific <- COVID19::covid19(country = "USA",
                                  level = 3,
                                  start = "2020-03-01",
                                  end = "2021-04-28")

# filter out NY data
ny_covid_data <- covid_data_specific %>%
  filter(administrative_area_level_3 == "New York City")

# Make plot of cases in NY
ny_covid_data_plot <- ny_covid_data %>%
  ggplot(aes(date, confirmed)) +
         geom_line() +
  labs(title = "Number of Confirmed Cases in New York City",
       y = "Confirmed Cases", x = "Date") +
  theme_clean()
  
ny_covid_data_plot

```
***Figure 2*** Looking specifically at New York City, we can see a similar trend. New York City was responsible for a large portion of the nations case numbers during the first wave of the pandemic.

```{r ny-zoomed-in}
# get specific covid data thats zoomed in on specific period
covid_data_specific_zoom <- COVID19::covid19(country = "USA",
                                  level = 3,
                                  start = "2020-03-01",
                                  end = "2020-04-10")

ny_covid_data_zoom <- covid_data_specific_zoom %>%
  filter(administrative_area_level_3 == "New York City")

# Make plot of cases in NY
ny_covid_data_zoom_plot <- ny_covid_data_zoom %>%
  ggplot(aes(date, confirmed)) +
         geom_line() +
  labs(title = "Number of Confirmed Cases in New York City",
       y = "Confirmed Cases", x = "Date") +
  geom_vline(xintercept = as.numeric(as.Date("2020-03-20")),
             linetype = "dashed",
             color = "green") +
  theme_clean()
  
ny_covid_data_zoom_plot
```
***Figure 3*** Going for a zoomed-in look, we can see how rapidly new cases of SARS-CoV-2 infection were being recorded during a short few-week period in New York City. This steep incline likely reflects a greater rate of testing. The dashed green line indicates when New York State issued a statewide mandatory stay at home order.

```{r ny-mobility-data}
# look at mobility data to get an idea of what mobility was like in NYC
covid_data_ny_mobility <- COVID19::covid19(country = "USA",
                              level = 3,
                              start = "2020-03-01",
                              end = "2020-04-10",
                              gmr =  paste0("https://www.gstatic.com/covid19/",
                                           "mobility/",
                                           "Global_Mobility_Report.csv"))

# filter NYC data
ny_covid_data_mobility <- covid_data_ny_mobility %>%
  filter(administrative_area_level_3 == "New York City")

# Plot
# uses code from https://designdatadecisions.wordpress.com/2015/07/23/
# html cont waterfall-plots-what-and-how/
ny_covid_data_mobility_plot <- ny_covid_data_mobility %>%
  ggplot(aes(date, transit_stations_percent_change_from_baseline)) +
  geom_bar(stat = "identity") +
  labs(title = "Bar Plot of Transit Mobility in New York City",
       x = "Date",
       y = "% change from baseline") +
  theme_clean()

ny_covid_data_mobility_plot

```
***Figure 4*** Bar plot showing percent change in mobility at transit stations in New York, from a baseline set before the pandemic. This metric contradicts the steep rise in cases during this time, unless you account for the variable incubation time of the SARS-CoV-2 virus, and the probability of higher case rates in early 2020 than shown above due to insufficient testing. For more mobility information, see Table 1. 

```{r vcf-clade-analysis}
# download clade file
clade_info <- read_tsv(paste0("https://raw.githubusercontent.com/nextstrain/",
                              "ncov/master/defaults/clades.tsv"))

# use clade file to do phylogenetic analysis of vcf data, using function 
# check_clades
clade_output <- make_sample_snps_table(clade_info, vcf_with_metadata)

# make plot
clade_plot <- heatmap(clade_output)

clade_plot
```
***Figure 5*** 
# Tables

```{r mobility-table}
# Make a table summarizing all of the google mobility data
# using code from https://stackoverflow.com/questions/40554231/
# html cont dplyr-lubridate-how-to-aggregate-a-dataframe-by-week/4055452
NY_covid_data_mobility %>%
  select(date, retail_and_recreation_percent_change_from_baseline,
         grocery_and_pharmacy_percent_change_from_baseline,
         parks_percent_change_from_baseline,
         transit_stations_percent_change_from_baseline,
         workplaces_percent_change_from_baseline,
         residential_percent_change_from_baseline) %>%
  mutate(week = week(date)) %>%
  group_by(week) %>%
  summarise(across(c(retail_and_recreation_percent_change_from_baseline,
                     grocery_and_pharmacy_percent_change_from_baseline,
                     parks_percent_change_from_baseline,
                     transit_stations_percent_change_from_baseline,
                     workplaces_percent_change_from_baseline,
                     residential_percent_change_from_baseline),
                   ~ mean(.x))) %>%
  knitr::kable(col.names = c("Week",
                             "Reail and Recreation",
                             "Grocery and Pharmacy",
                             "Parks",
                             "Transit Stations",
                             "Workplaces",
                             "Residential"))

```
***Table 1*** Summary of Google mobility data from weeks 9-15 of the year 2020. Values represent the mean percent change from baseline for that week. The table shows a gradual decrease in most mobility factors over the first several weeks of the pandemic. Notably, the Google data shows an increase in mobility to grocery stores during weeks 9-11 which likely reflects the panic fueled stockpiling of necessities, a trend seen across the US which led to a shortage of items like toilet paper. Additionally, there was increasing mobility in residential areas, reflecting people seeking outdoor exercise around their homes during quarantine.   

```{r variant-table}
# make a table showing different SNPs and their locatioms
vcf_with_metadata %>%
  group_by(gene) %>%
  tally() %>% 
  knitr::kable(col.names = c("Gene Name",
                             "Count"))
```
***Table 2***

```{r vcf-table}
# make a table showing different SNPs and their locations
vcf_with_metadata %>%
  filter(!is.na(gene)) %>% # get rid of SNPs not in gene regions
  group_by(gene, pos) %>%
  tally() %>% # this gives a column n for the number of gene by position
  group_by(gene) %>%
  tally() %>%
  knitr::kable(col.names = c("Gene Name",
                             "Count"))
```
***Table 3*** Count of distinct SNP's found for each named gene in the SARS-CoV-2 genome. 

```{r example-table}
# An example table to show the length of each gene using its start and end
gene_table %>%
  mutate(length = end - start) %>%
  select(gene_name, start, end, length) %>%
  knitr::kable(col.names = c("Gene Name",
                             "Start",
                             "End",
                             "Length"))
```
**Table 4**: Gene names, locations, and lengths in the SARS-CoV-2 genome. 
# Sources Cited
