---
title: 'Analysis of SARS-CoV-2 Infection in New York City in the Early Stages of the COVID-19 Pandemic'
author: "Maggie Walsh"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output: pdf_document
csl: bioinformatics.csl
bibliography: references.bib
params:
  gff_file_path: "data/metadata/sars_refgenome_annotation.gff"
  vcf_dir_path: "data/11_vcf_output_for_R"
  sra_runtable_path: "data/00_sra_runtable/SraRunTable_NY_fixed.txt"
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE,
                      warning = FALSE,
                      message = FALSE,
                      error = TRUE)
```

# Background and Overview

In early 2020, the world was forever changed by the spread of the coronavirus disease (COVID-19), which was declared a global pandemic by the WHO in early March. New York City quickly emerged as the epicenter in the United States with 203,000 confirmed cases between the months of March and May [@thompson2020covid]. Although news of the virus was widely overlooked initially, people quickly became aware of the insidious nature of the SARS-CoV-2 virus. The SARS-CoV-2 virus was hard to track due to it's variable incubation time and it's ability to spread via asymptomatic carriers. The only way to fight this virus early on was through limiting person to person interaction. New York City was placed under a statewide lock-down on March 22, 2020. The number of confirmed cases peaked a week later [@thompson2020covid]. As the machinery of our society was grinding to a halt, researchers raced to understand the SARS-CoV-2 virus in order to prevent further devastation. My goal for this report was to synthesize some of that research, and generate my own analysis that explains what was going on in New York City during the onset of the pandemic. My analysis started with a look into SARS-CoV-2 case numbers in New York, followed by a look at changes in population mobility during this time. I also used a set of bioinformatics tools to analyze sequence data taken from nasopharyngeal swabs of people in New York City from early March to late April. Twelve samples were sequenced and variants were called against a reference genome to determine SNPs for analysis. The common consensus among researchers is that the high case numbers in New York City was the result of uncontrolled community spread after many different travelers (mainly from Europe) introduced the virus to the population [@bushman2020detection; @gonzalez2020introductions; @maurano2020sequencing]. My findings corroborate this story. I found that the population data pointed to  

# Methods

## Sample Sequencing and Variant Calling
I downloaded sequences to a remote server from the [NCBI database](https://www.ncbi.nlm.nih.gov/) using the [accession ID](https://www.ncbi.nlm.nih.gov//bioproject/PRJNA721724), utilizing [fasterq-dump](https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump) from the NCBI SRA toolkit. Samples were trimmed using the trimmomatic tool [@bolger2014trimmomatic]. My workflow utilized the Burrows Wheeler Aligner (BWA) to map reads against the refrence genome [@li2009fast] and SAMtools and BCFtools to sort and process the mapped reads [@li2009sequence]. Specific steps of my pipeline can be found in the [code folder](https://github.com/walshmags/walshmags-bioinformatics-final-project/tree/main/code) of this repository. Parts of the pipeline approach are based on the pipeline described in the [Data Carpentry Genomics lessons](https://datacarpentry.org/genomics-workshop/), which are made available under a [CC-BY 4.0 license](https://creativecommons.org/licenses/by/4.0/). I executed the bash pipeline on the USF server using a makefile to output vcf files for analysis.

## Analysis

### Reading in vcf files
I analyzed the output from the bash pipeline in RStudio using a series of functions designed to format the vcf files into tabular data. The vcf files were read into R using the vcfR package [@knaus2017vcfr].

### Analysis of tabular data
I manipulated tabular data using the [dplyr](https://dplyr.tidyverse.org/) package. I made figures using the package [ggplot2](https://ggplot2.tidyverse.org/reference/index.html) with additional formatting using the packages [ggthemes](https://cran.r-project.org/web/packages/ggthemes/index.html) and [gghighlight](https://cran.r-project.org/web/packages/gghighlight/vignettes/gghighlight.html).

### Additional data sources
I accessed additional data sources using the COVID19 R package [@Guidotti2020]. Case numbers originated from the Oxford [COVID-19 GOVERNMENT RESPONSE TRACKER](https://www.bsg.ox.ac.uk/research/research-projects/covid-19-government-response-tracker) data set. Mobility data was from the [Google Mobility Reports](https://www.google.com/covid19/mobility/). I also used SARS-CoV-2 clade definitions from the [NextClade](https://clades.nextstrain.org/) project.

# Results

## Case Numbers
I started my assessment of case numbers by looking through a wider lens. I wanted to look at New York State as a whole, and compare its confirmed case numbers to the rest of the country. As seen in Figure 1, the state of New York was far beyond other states in terms of positive case numbers early on in the pandemic. However, by the Summer of 2020 it seemed that the exponential growth in cases was stalled, and New York was no longer the state with the most confirmed cases by August. However, case numbers in New York did skyrocket again in the Winter months. To get an exact look at New York City, I narrowed my scope (Figure 2). It appears that the trends in case numbers in New York City followed a very similar pattern as seen in the state as a whole. Next, I narrowed in on my period of interest, the time frame in which my samples were collected (Figure 3). Between March 1st and April 10th, case numbers increased at alarming rates, going from almost no confirmed cases to tens of thousands of cases in a matter of weeks. This sharp incline was seen even after officials ordered a statewide lock-down.

## Mobility Reports
I wanted to use mobility data to gauge how New York City residents were adapting during this time. I chose to visualize transit mobility because I know that a large proportion of the population uses public transit to get around (Figure 4). Transit mobility experienced a sharp decline between March and April of 2020, diving over 80% below baseline levels by mid April. A similar trend can be seen in almost all of the mobility categories (Table 1). Interestingly, there is a clear inverse relationship between case numbers and citizen mobility.

## Variant Analysis
For my variant analysis, I compiled a list of distinct SNPs for each named gene in the SARS-CoV-2 genome (Table 2). I found a total of 7 different distinct SNPs in protein-coding regions. Three of those were found in the nucleocapsid (N) gene, while one distinct SNP was found in each of the remaining genes, S, M, ORF3a, and ORF10. To determine the significance of the variants found in each of my samples, I generated a matrix showing similarity between each sample and each of the known SARS-CoV-2 clades (Table 3). I used this information to generate a heat map which visualizes these similarities (Figure 5). The heat map singles out two samples (SRR14232249 and SRR14232250) as genetically distinct from the rest, with more similarity to the 20B and 20I/501Y.V1 clades. The remaining samples showed more similarity to the 20C and 20H/501Y.V2 clades. It is important to note that clade 20I/501Y.V1 is a descendant of clade 20B, and clade 20H/501Y.V2 is a descendant of clade 20C [@bedford_hodcroft_neher_2021]. This means that samples that are similar to 20B or 20C will also show some similarity to their descendants.

# Discussion

## Putting Case Numbers and Mobility Reports Into Context
Many people were shocked at news reports which displayed the soaring numbers of confirmed SARS-CoV-2 infections in New York City. However, in depth analysis reveals that this may not be so shocking. The first factor that we must consider is the availability of tests early on in the pandemic [@thompson2020covid]. It is possible that the rapid incline in positive case numbers (seen in figures 1-3) was more due to more tests being performed than an increased prevalence of the virus. Research suggests that the virus was already prevalent in places like New York City when rapid testing began. Multiple research studies have indicated that the SARS-CoV-2 virus was circulating in New York and other US cities long before public health officials originally thought [@bushman2020detection; @gonzalez2020introductions]. There is even evidence that suggests that by March of 2020, new infections were being introduced via domestic spread between US cities [@fauver2020coast]. It is plausible that while going unnoticed, the SARS-CoV-2 virus was able to spread widely and rapidly, and was brought to New York City a multitude of times due to its status as a major travel hub. These viral introductions were most likely able to easily spread via community infection due to the heightened population density associated with the urban environment of New York City [@rocklov2020high]. Due to the stealthy nature of this virus, it is plausible that asymptomatic or mildly ill individuals could infect others without alarm [@oran2020prevalence]. Furthermore, the incubation period of SARS-CoV-2 has been reported as variable, with an average of about 5 days until someone begins to feel symptoms, and it may take even longer for someone to begin to experience serious illness due to Covid-19 [@wu2020sars]. These factors help to explain how SARS-CoV-2 was spreading early on, and why case numbers soared in New York in March, even after physical distancing initiatives were put in place.

## Using Variant Analysis to Explain the Outbreak
Variant analysis can be an efficient way of tracking the origins of particular viral disease outbreaks. I used variants detected in my samples to detect SARS-CoV-2 genomes genetically related to two distinct clade lineages, the 20B lineage and the 20C lineage. Both lineages were circulating in Europe at the period of interest. This suggests a link between the strains of SARS-CoV-2 in Europe and those known to have caused some of the early infections in New York City. My findings echo the results of other phylogenetic analyses [@bushman2020detection; @gonzalez2020introductions; @maurano2020sequencing].  

```{r load-packages-and-functions}
library("vcfR")
library("ggplot2")
library("dplyr")
library("ggthemes")
library("COVID19")
library("gghighlight")
library("lubridate")
library("readr")

# this loads all function files in the code/functions directory
function_paths <- list.files("code/functions", full.names = TRUE)
purrr::walk(function_paths, source)
```

```{r load-vcf-data}
# load in, tidy, and stack all of the VCF files in the target directory
# these are the output of the bash pipeline
stacked_vcfs <- parse_tidy_and_stack_vcfs(
  vcf_dir_path = params$vcf_dir_path)

# load in the gff file with genome annotations for SARS-CoV-2
gff <- read_gff(gff_file_path = params$gff_file_path)

# pull out just the named genes from the annotation file
gene_table <- extract_genes_from_gff(annotation_object = gff)

# combine the stacked vcf table with the gene names and the metadata from
# the sra runtable file
vcf_with_metadata <- add_genes_metadata_to_vcfstack(
  sra_runtable_path = params$sra_runtable_path,
  stacked_vcf = stacked_vcfs,
  cleaned_genes_table = gene_table)
```

# Figures

```{r Covid-Data}
#trying to use COVID19 package to look at data regarding New York city
# first get all covid data from period of interest
covid_data_broad <- COVID19::covid19(country = "USA",
                                  level = 2,
                                  start = "2020-02-01",
                                  end = "2021-04-28")

# make a plot summarizing covid cases by state during this time
covid_plot <- covid_data_broad %>%
  ggplot(aes(date, confirmed)) +
  geom_line(aes(group = administrative_area_level_2)) +
  gghighlight(administrative_area_level_2 == "New York") +
  labs(title = "SARS-CoV-2 Cases by State",
       x = "Date", y = "Confirned Cases") +
  theme_clean()

covid_plot

```
***Figure 1***: Plot of confirmed SARS-CoV-2 cases in the past year. New York state was the hardest hit early on in the pandemic having a majority of the nations confirmed cases. The steep rise in cases was contained in subsequent months. In recent winter months, New York, alongside other states, saw another steep rise in cases.


```{r NY-Data}
# get specific covid data
covid_data_specific <- COVID19::covid19(country = "USA",
                                  level = 3,
                                  start = "2020-03-01",
                                  end = "2021-04-28")

# filter out NY data
ny_covid_data <- covid_data_specific %>%
  filter(administrative_area_level_3 == "New York City")

# Make plot of cases in NY
ny_covid_data_plot <- ny_covid_data %>%
  ggplot(aes(date, confirmed)) +
         geom_line() +
  labs(title = "Number of Confirmed Cases in New York City",
       subtitle = "over the past year",
       y = "Confirmed Cases", x = "Date") +
  theme_clean()
  
ny_covid_data_plot

```
***Figure 2*** Looking specifically at New York City, we can see a similar trend as the state case numbers shown above. New York City was responsible for a large portion of the nations case numbers during the first wave of the pandemic.

```{r ny-zoomed-in}
# get specific covid data thats zoomed in on specific period
covid_data_specific_zoom <- COVID19::covid19(country = "USA",
                                  level = 3,
                                  start = "2020-03-01",
                                  end = "2020-04-10")

ny_covid_data_zoom <- covid_data_specific_zoom %>%
  filter(administrative_area_level_3 == "New York City")

# Make plot of cases in NY
ny_covid_data_zoom_plot <- ny_covid_data_zoom %>%
  ggplot(aes(date, confirmed)) +
         geom_line() +
  labs(title = "Number of Confirmed Cases in New York City",
       subtitile = "early March to mid April",
       y = "Confirmed Cases", x = "Date") +
  geom_vline(xintercept = as.numeric(as.Date("2020-03-20")),
             linetype = "dashed",
             color = "gray") +
  theme_clean()
  
ny_covid_data_zoom_plot
```
***Figure 3*** Going for a zoomed-in look, we can see how rapidly new cases of SARS-CoV-2 infection were being recorded during a short few-week period in New York City. This steep incline likely reflects a greater rate of testing. The dashed line indicates when New York State issued a statewide mandatory stay at home order.

```{r ny-mobility-data}
# look at mobility data to get an idea of what mobility was like in NYC
covid_data_ny_mobility <- COVID19::covid19(country = "USA",
                              level = 3,
                              start = "2020-03-01",
                              end = "2020-04-10",
                              gmr =  paste0("https://www.gstatic.com/covid19/",
                                           "mobility/",
                                           "Global_Mobility_Report.csv"))

# filter NYC data
ny_covid_data_mobility <- covid_data_ny_mobility %>%
  filter(administrative_area_level_3 == "New York City")

# Plot
# uses code from https://designdatadecisions.wordpress.com/2015/07/23/
# html cont waterfall-plots-what-and-how/
ny_covid_data_mobility_plot <- ny_covid_data_mobility %>%
  ggplot(aes(date, transit_stations_percent_change_from_baseline)) +
  geom_bar(stat = "identity") +
  labs(title = "Transit Mobility in New York City",
       x = "Date",
       y = "% change from baseline") +
  theme_clean()

ny_covid_data_mobility_plot

```
***Figure 4*** Bar plot showing percent change in mobility at transit stations in New York, from a baseline set before the pandemic. This metric contradicts the steep rise in cases during this time, unless you account for the variable incubation time of the SARS-CoV-2 virus, and the probability of higher case rates in early 2020 than shown above due to insufficient testing. For more mobility information, see Table 1. 

```{r vcf-clade-analysis}
# download clade file
clade_info <- read_tsv(paste0("https://raw.githubusercontent.com/nextstrain/",
                              "ncov/master/defaults/clades.tsv"))

# use clade file to do phylogenetic analysis of vcf data, using function 
# check_clades
clade_output <- make_sample_snps_table(clade_info, vcf_with_metadata)

# make plot
clade_plot <- 
  heatmap(clade_output,
          margins = c(9,8),
          na.rm = TRUE,
          main = "Sample Similarity to SARS-CoV-2 Clades",
          xlab = "Run ID",
          ylab = "Clade")

clade_plot
```
***Figure 5*** Heat map showing relationships between clades and sample genotypes. Darker colors represent a greater similarity. Hierarchical clustering points to two genetically distinct groups, with differing clade distinctions. For exact values, see Table 3.

# Tables

```{r mobility-table}
# Make a table summarizing all of the google mobility data
# using code from https://stackoverflow.com/questions/40554231/
# html cont dplyr-lubridate-how-to-aggregate-a-dataframe-by-week/4055452
NY_covid_data_mobility %>%
  select(date, retail_and_recreation_percent_change_from_baseline,
         grocery_and_pharmacy_percent_change_from_baseline,
         parks_percent_change_from_baseline,
         transit_stations_percent_change_from_baseline,
         workplaces_percent_change_from_baseline,
         residential_percent_change_from_baseline) %>%
  mutate(week = week(date)) %>%
  group_by(week) %>%
  summarise(across(c(retail_and_recreation_percent_change_from_baseline,
                     grocery_and_pharmacy_percent_change_from_baseline,
                     parks_percent_change_from_baseline,
                     transit_stations_percent_change_from_baseline,
                     workplaces_percent_change_from_baseline,
                     residential_percent_change_from_baseline),
                   ~ mean(.x))) %>%
  knitr::kable(col.names = c("Week",
                             "Reail and Recreation",
                             "Grocery and Pharmacy",
                             "Parks",
                             "Transit Stations",
                             "Workplaces",
                             "Residential"))

```
***Table 1*** Summary of Google mobility data from weeks 9-15 of the year 2020. Values represent the mean percent change from baseline for that week. The table shows a gradual decrease in most mobility factors over the first several weeks of the pandemic. Notably, the Google data shows an increase in mobility to grocery stores during weeks 9-11 which likely reflects the panic fueled stockpiling of necessities, a trend seen across the US which led to a shortage of items like toilet paper. Additionally, there was increasing mobility in residential areas, likely a reflection of people seeking outdoor exercise around their homes during quarantine.   

```{r vcf-table}
# make a table showing different SNPs and their locations
vcf_with_metadata %>%
  filter(!is.na(gene)) %>% # get rid of SNPs not in gene regions
  group_by(gene, pos) %>%
  tally() %>% # this gives a column n for the number of gene by position
  group_by(gene) %>%
  tally() %>%
  knitr::kable(col.names = c("Gene Name",
                             "Count"))
```
***Table 2*** Count of distinct SNPs found for each named gene in the SARS-CoV-2 genome. In the samples provided, there were 7 distinct SNPs detected in named genes. Three of those variants were found in the N gene. It is worth noting that the SNP found in the spike protein was identified as the D614G variant which was the first known variant to surpass the wild type allele to become globally dominant.

```{r clade-table}
# make a table showing results from check clades function
knitr::kable(clade_output, digits = 2)

```
***Table 3*** Matrix showing relationship between samples and clades. Value is the proportion of of SNPs in common over the total number in each clade. Higher values correspond to a greater similarity.

# Sources Cited
