---
title: 'Analysis of SARS-CoV-2 Infection In New York City at Early Stages of the COVID-19 Pandemic'
author: "Maggie Walsh"
date: "`r format(Sys.time(), '%d %B, %Y')`"
output: pdf_document
csl: bioinformatics.csl
bibliography: references.bib
params:
  gff_file_path: "data/metadata/sars_refgenome_annotation.gff"
  vcf_dir_path: "data/11_vcf_output_for_R"
  sra_runtable_path: "data/00_sra_runtable/SraRunTable_NY_fixed.txt"
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE,
                      warning = FALSE,
                      message = FALSE,
                      error = TRUE)
```

# Background and Overview

This is a report on SARS-CoV-2, including some variant analysis [@koyama2020variant].

# Methods

See the set of tutorials on the [vcfR package website](https://knausb.github.io/vcfR_documentation/index.html).

You may also want to use any of a range of different COVID data packages and data sources:

* https://kjhealy.github.io/covdata/
* https://github.com/como-ph/oxcovid19
* https://ropensci.org/blog/2020/10/20/searching-medrxivr-and-biorxiv-preprint-data/
* https://covidtracking.com/data/api
  * `readr::read_csv("https://api.covidtracking.com/v1/states/daily.csv")`

## Subsections are ok too

# Results and Discussion

```{r load-packages-and-functions}
library("vcfR")
library("ggplot2")
library("dplyr")
library("ggthemes")
library("COVID19")
library("gghighlight")
library("lubridate")

# this loads all function files in the code/functions directory
function_paths <- list.files("code/functions", full.names = TRUE)
purrr::walk(function_paths, source)
```

```{r load-vcf-data}
# load in, tidy, and stack all of the VCF files in the target directory
# these are the output of the bash pipeline
stacked_vcfs <- parse_tidy_and_stack_vcfs(
  vcf_dir_path = params$vcf_dir_path)

# load in the gff file with genome annotations for SARS-CoV-2
gff <- read_gff(gff_file_path = params$gff_file_path)

# pull out just the named genes from the annotation file
gene_table <- extract_genes_from_gff(annotation_object = gff)

# combine the stacked vcf table with the gene names and the metadata from
# the sra runtable file
vcf_with_metadata <- add_genes_metadata_to_vcfstack(
  sra_runtable_path = params$sra_runtable_path,
  stacked_vcf = stacked_vcfs,
  cleaned_genes_table = gene_table)
```

# Figures

```{r sample-plot}
# create a plot of unique SNP locations within each gene across all samples
vcf_with_metadata %>%
  filter(!is.na(gene)) %>% # get rid of SNPs not in gene regions
  group_by(gene, pos) %>%
  tally() %>% # this gives a column n for the number of gene by position
  group_by(gene) %>%
  tally() %>% # this collapses that down to the number of unique SNP locations
  ggplot(aes(x = gene,
             y = n)) +
    geom_col() +
    labs(title = "Count of distinct SNPs in Named SARS-CoV-2 Genes",
         x = "Gene Name") +
  theme_few() # get rid of the grey background
```

**Figure 1**: N and S genes have more unique SNPs in the set of samples analyzed.

```{r Covid-Data}
#trying to use COVID19 package to look at data regarding New York city
# first get all covid data from period of interest
covid_data_broad <- COVID19::covid19(country = "USA",
                                  level = 2,
                                  start = "2020-02-01",
                                  end = "2021-04-28")

# make a plot summarizing covid cases by state during this time
covid_plot <- covid_data_broad %>%
  ggplot(aes(date, confirmed)) +
  geom_line(aes(group = administrative_area_level_2)) +
  gghighlight(administrative_area_level_2 == "New York") +
  labs(title = "Number of Confirmed SARS-CoV-2 Cases by State",
       x = "Date", y = "Confirned Cases") +
  theme_clean()

covid_plot

```
***Figure 1***: Plot of confirmed SARS-CoV-2 cases in the past year. New York state was the hardest hit early on in the pandemic having a majority of the nations confirmed cases. The steep rise in cases was contained in subsequent months. In recent winter months, New York, alongside other states, has again seen a steep rise in cases.


```{r NY-Data}
# get specific covid data
covid_data_specific <- COVID19::covid19(country = "USA",
                                  level = 3,
                                  start = "2020-03-01",
                                  end = "2021-04-28")

# filter out NY data
ny_covid_data <- covid_data_specific %>%
  filter(administrative_area_level_3 == "New York City")

# Make plot of cases in NY
ny_covid_data_plot <- ny_covid_data %>%
  ggplot(aes(date, confirmed)) +
         geom_line() +
  labs(title = "Number of Confirmed Cases in New York City",
       y = "Confirmed Cases", x = "Date") +
  theme_clean()
  
ny_covid_data_plot

```
***Figure 2*** Looking specifically at New York City, we can see a similar trend. New York City was responsible for a large portion of the nations case numbers during the first wave of the pandemic.

```{r ny-zoomed-in}
# get specific covid data thats zoomed in on specific period
covid_data_specific_zoom <- COVID19::covid19(country = "USA",
                                  level = 3,
                                  start = "2020-03-01",
                                  end = "2020-04-10")

ny_covid_data_zoom <- covid_data_specific_zoom %>%
  filter(administrative_area_level_3 == "New York City")

# Make plot of cases in NY
ny_covid_data_zoom_plot <- ny_covid_data_zoom %>%
  ggplot(aes(date, confirmed)) +
         geom_line() +
  labs(title = "Number of Confirmed Cases in New York City",
       y = "Confirmed Cases", x = "Date") +
  geom_vline(xintercept = as.numeric(as.Date("2020-03-20")),
             linetype = "dashed",
             color = "green") +
  theme_clean()
  
ny_covid_data_zoom_plot
```
***Figure 3*** Going for a zoomed-in look, we can see how rapidly new cases of SARS-CoV-2 infection were being recorded during a short few-week period in New York City. This steep incline likely reflects a greater rate of testing. The dashed green line indicates when New York State issued a statewide mandatory stay at home order.

```{r ny-mobility-data}
# look at mobility data to get an idea of what mobility was like in NYC
covid_data_ny_mobility <- COVID19::covid19(country = "USA",
                              level = 3,
                              start = "2020-03-01",
                              end = "2020-04-10",
                              gmr =  paste0("https://www.gstatic.com/covid19/",
                                           "mobility/",
                                           "Global_Mobility_Report.csv"))

# filter NYC data
ny_covid_data_mobility <- covid_data_ny_mobility %>%
  filter(administrative_area_level_3 == "New York City")

# Plot
# uses code from https://designdatadecisions.wordpress.com/2015/07/23/
# html cont waterfall-plots-what-and-how/
ny_covid_data_mobility_plot <- ny_covid_data_mobility %>%
  ggplot(aes(date, transit_stations_percent_change_from_baseline)) +
  geom_bar(stat = "identity") +
  labs(title = "Bar Plot of Transit Mobility in New York City",
       x = "Date",
       y = "% change from baseline") +
  theme_clean()

ny_covid_data_mobility_plot

```
***Figure 4*** Bar plot showing percent change in mobility at transit stations in New York, from a baseline set before the pandemic. This metric contradicts the steep rise in cases during this time, unless you account for the variable incubation time of the SARS-CoV-2 virus, and the probability of higher case rates in early 2020 than shown above due to insufficient testing. For more mobility information, see Table 1. 

```{r vcf-analysis}
# use vcfr package to do analysis of vcf data
vcf_with_metadata %>%
  filter(!is.na(gene)) %>% # get rid of SNPs not in gene regions
  group_by(Collection_Date, gene) %>%
  ggplot(aes(Collection_Date)) +
    geom_bar(position = "dodge2", aes(fill = gene)) +
    labs(title = "Functional Variants Found in Sample by Collection Date",
         x = "Day") +
  theme_clean()

```

# Tables

```{r example-table}
# An example table to show the length of each gene using its start and end
gene_table %>%
  mutate(length = end - start) %>%
  select(gene_name, start, end, length) %>%
  knitr::kable(col.names = c("Gene Name",
                             "Start",
                             "End",
                             "Length"))
```
**Table 1**: Gene names, locations, and lengths in the SARS-CoV-2 genome. Higher SNP counts in the S and N genes may be related to the larger size of these genes.

```{r mobility-table}
# Make a table summarizing all of the google mobility data
NY_covid_data_mobility %>%
  select(date, retail_and_recreation_percent_change_from_baseline,
         grocery_and_pharmacy_percent_change_from_baseline,
         parks_percent_change_from_baseline,
         transit_stations_percent_change_from_baseline,
         workplaces_percent_change_from_baseline,
         residential_percent_change_from_baseline) %>%
  mutate(week = week(date)) %>%
  group_by(week) %>%
  summarise(across(c(retail_and_recreation_percent_change_from_baseline,
                     grocery_and_pharmacy_percent_change_from_baseline,
                     parks_percent_change_from_baseline,
                     transit_stations_percent_change_from_baseline,
                     workplaces_percent_change_from_baseline,
                     residential_percent_change_from_baseline),
                   ~ mean(.x))) %>%
  knitr::kable(col.names = c("Week",
                             "Reail and Recreation",
                             "Grocery and Pharmacy",
                             "Parks",
                             "Transit Stations",
                             "Workplaces",
                             "Residential"))

```
***Table 1*** Summary of Google mobility data from weeks 9-15 of the year 2020. Values represent the mean percent change from baseline for that week. The table shows a gradual decrease in most mobility factors over the first several weeks of the pandemic. Notably, the Google data shows an increase in mobility to grocery stores during weeks 9-11 which likely reflects the panic fueled stockpiling of necessities, a trend seen across the US which led to a shortage of items like toilet paper. Additionally, there was increasing mobility in residential areas, reflecting people seeking outdoor exercise around their homes during quarantine.   


```{r vcf-table}
# make a table showing different SNPs and their locations
vcf_with_metadata %>%
  filter(!is.na(gene)) %>% # get rid of SNPs not in gene regions
  group_by(gene, pos) %>%
  tally() %>% # this gives a column n for the number of gene by position
  group_by(gene) %>%
  tally() %>%
  knitr::kable(col.names = c("Gene Name",
                             "Count"))
```
***Table 2*** Count of distinct SNP's found for each named gene in the SARS-CoV-2 genome. 
# Sources Cited
